src/map/services/index.ts

```diff
@@ -51,9 +125,7 @@ export async function getMapLayers() {
     }
   }`
 
-  const { results: mapLayerResults } = await fetchMap('mapLayerSearch', query)
-
-  return mapLayerResults
+  return (await fetchMap<MapLayer>('mapLayerSearch', query)).results
```

Comment:
just in case the request fails?

```suggestion
  return (await fetchMap<MapLayer>('mapLayerSearch', query))?.results 
```